zktest/src/archive/test2/F86-ZK-3961.zul

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+F86-ZK-3961.zul
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Thu Jul 12 15:51:26 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+
+-->
+
+<zk>
+	<zscript>
+		public void setHeight300() {
```

Comment:
You can create a String parameter and pass the 300px, 100px and null. Thus you just only need one method.

Reply:
ok done